### PR TITLE
[Storage] Remove track1 SDK from Queue cmdlets

### PR DESCRIPTION
**Pull request by yifanz7**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [x] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Storage/Storage/Common/AzureStorageQueue.cs [hunk 1]
```diff
@@ -48,27 +37,10 @@ public class AzureStorageQueue : AzureStorageBase
...
-            {
-                if (privateQueueClient == null)
-                {
-                    privateQueueClient = GetTrack2QueueClient(this.CloudQueue, (AzureStorageContext)this.Context);
-                }
-                return privateQueueClient;
-            }
-        }
-        private QueueClient privateQueueClient = null;
+        public QueueClient QueueClient { get; private set; }
```
Comment (yifanz7) [inline src/Storage/Storage/Common/AzureStorageQueue.cs:43]: Add format for QueueClient in xml files. Delete QueueClient format

File: src/Storage/Storage/Storage.generated.format.ps1xml [hunk 2]
```diff
@@ -76,16 +72,12 @@
...
-                <Alignment>Left</Alignment>
-                <ScriptBlock>$_.EncodeMessage</ScriptBlock>
-              </TableColumnItem>
             </TableColumnItems>
           </TableRowEntry>
         </TableRowEntries>
       </TableControl>
       <GroupBy>
-        <ScriptBlock>$_.CloudQueue.ServiceClient.BaseUri</ScriptBlock>
+        <ScriptBlock>$_.QueueClient.Uri</ScriptBlock>
```
Comment (blueww) [inline src/Storage/Storage/Storage.generated.format.ps1xml:80]: Same, Here should not be the queue Uri, but the storage account Queue endpoint. The goal is to make the queue from same account be in a group. 
If it's not easy to get the account queue endpoint, you can also try to group the queue with account name. 

Currently, each queue has a group, the PSH console output is not easy to read.

File: src/Storage/Storage/Storage.format.ps1xml [hunk 3]
```diff
@@ -269,12 +269,12 @@
...
     <View>
-      <Name>Microsoft.Azure.Storage.Queue.CloudQueue</Name>
+      <Name>Azure.Storage.Queues.QueueClient</Name>
       <ViewSelectedBy>
-        <TypeName>Microsoft.Azure.Storage.Queue.CloudQueue</TypeName>
+        <TypeName>Azure.Storage.Queues.QueueClient</TypeName>
       </ViewSelectedBy>
       <GroupBy>
-        <ScriptBlock>$_.ServiceClient.BaseUri</ScriptBlock>
+        <ScriptBlock>$_.QueueClient.Uri</ScriptBlock>
```
[the remark this answers is not in the document]
Comment (yifanz7) [reply src/Storage/Storage/Storage.format.ps1xml]: updated to account name.

File: src/Storage/Storage/Storage.generated.format.ps1xml (hunk 2 above)
Comment (yifanz7) [reply src/Storage/Storage/Storage.generated.format.ps1xml]: updated to account name.

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**